src/distrs/beta.jl

```diff
@@ -1,19 +1,71 @@
 # functions related to beta distributions
 
+using HypergeometricFunctions: _₂F₁
+
 import .RFunctions:
-    betapdf,
-    betalogpdf,
-    betacdf,
-    betaccdf,
-    betalogcdf,
-    betalogccdf,
-    betainvcdf,
-    betainvccdf,
+    # betapdf,
+    # betalogpdf,
+    # betacdf,
+    # betaccdf,
+    # betalogcdf,
+    # betalogccdf,
+    # betainvcdf,
+    # betainvccdf,
```

Comment:
I wonder why these lines are only commented out and not removed completely?

Reply:
I thought it was useful to keep them around while we are working on removing all of them. I'm also fine with deleting them, though.